lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
```

Comment:
But yeah, I think we can audit existing PE vanagon projects (there aren't that many) to make sure target_repo is set to *something*, it's probably a bug not a feature that those ended up in products before :)